code/web/src/modules/user/ProfileEdit.js

```diff
@@ -35,21 +31,20 @@ class ProfileEdit extends Component {
     this.state = {
       isLoading: false,
       userDetails: {
-        // id: 0,
         name: this.props.userDetails.name,
         email: this.props.userDetails.email,
-        streetAddress: "",
-        cityAddress: "",
-        addressState: "",
-        zip: "",
-        description: "",
-        image: "",
+        streetAddress: this.props.userDetails.streetAddress,
```

Comment:
I think this works, setting the form state to props, and then updating in the inputs, but I also think the inputs could initially be set to these values.